spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -6,15 +6,17 @@
 
   before(:each) do
     allow(MiqServer).to receive(:my_zone).and_return("default")
-    hostname = 'host.example.com'
-    token = 'theToken'
+    # env vars for easier VCR recording, see test_objects_record.sh
+    hostname = ENV["API_HOST"] || "host.example.com"
+    token = ENV["API_TOKEN"] || "theToken"
```

Comment:
We have some automation around cluster creation and there these are named:
```
OPENSHIFT_MASTER_HOST
OPENSHIFT_MANAGEMENT_ADMIN_TOKEN
```
IT could prove useful to have the same names, up to you.

Reply:
https://github.com/container-mgmt/cm-ops-flow/tree/a455c59f7cc7d00a6e534de570de3dbe24a53e4b#needed-environment-variables